server/config/sessions/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  MONGO: require('./mongo'),
```

Comment:
@mjewell I couldn't `[DB_TYPES.MONGO] : require('./mongo')` as you suggested (was returning undefined).

Not sure if this is because we need to compile it with babel beforehand? [Example here](https://babeljs.io/repl/#?evaluate=true&presets=es2015&experimental=false&loose=false&spec=false&code=var%20DB_TYPES%20%3D%20%7B%0A%20%20%20%20MONGO%3A%20'MONGO'%2C%0A%20%20%20%20POSTGRES%3A%20'POSTGRES'%2C%0A%20%20%20%20NONE%3A%20'NONE'%0A%7D%3B%0A%0Avar%20a%20%3D%20%7B%0A%20%20%5BDB_TYPES.MONGO%5D%3A%20'b'%0A%7D%3B%0Aconsole.log(a)) @psimyn Thoughts?

I did the simple MONGO just to get it over the line. If it s a matter of ES6, it might be worth kicking off #26